.devcontainer/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.162.0/containers/typescript-node/.devcontainer/base.Dockerfile
```

Comment:
This file was automatically created via https://docs.github.com/en/github/developing-online-with-codespaces/configuring-codespaces-for-your-project#using-a-pre-built-container-configuration with the "Node.js and TypeScript" option, wasn't sure which version to pick so I went with the default (`14`)

Reply:
`14` is perfect 👍